docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   gcs:
-    image: fsouza/fake-gcs-server
+    image: simonbohnen/fake-gcs-server
```

Comment:
Could you open a PR to get this merged upstream?

(Not blocking to get this PR merged for me, though).

Reply:
I'm rather thinking that the bug is on the side of the client SDK, see my comment [here](https://github.com/googleapis/google-resumable-media-python/issues/22#issuecomment-1263971993). I chose to modify the server instead as integrating that into our tests is easier than changing the modified client, I think.

I'm going to open a PR for the client in addition to the comment I posted.